acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -192,7 +194,7 @@ export default function init(element) {
   dcScript.dataset.load_imslib = 'false';
   dcScript.dataset.enable_unload_prompt = 'true';
   if (preRenderDropZone) {
-    dcScript.dataset.pre_rendered = 'true';
+    dcScript.dataset.pre_rendered = 'true'; // TODO: remove this line
```

Comment:
why?
or when?

Reply:
until we update the widget, we'll need this bit to do SSR with the old widget.  Going forward, it won't be needed anymore.  The widget will check for the bit on the CID element.